unit-tests/dds/test-metadata-syncer.py

```diff
@@ -339,5 +339,44 @@ def md_thread():
         test.check_equal( md_id( last_metadata() ), 9 )
     test.check_equal( len(dropped_metadata), 0 )
 
+with test.closure( 'Two threads, slow callback -> different callbacks intervined' ):
```

Comment:
intervined? Not sure this is clear...

Reply:
I don't mind changing the description. Need to be on the lines of "one callback starts when the previous is still running"